de-lib/src/main/java/org/iplantc/de/theme/base/client/diskResource/DiskResourceMessages.properties

```diff
@@ -87,3 +86,18 @@ overWiteMetadata = Some metadata already exists. Do you want to overwrite them ?
 doiRequestFail =  DOI Request failed!
 doiRequestSuccess = Your request for DOI has been submitted! You will receive notification when there are updates to this request.
 fileUploadsSuccess = The file(s) {0,list} were successfully uploaded.
+inputLbl = Select folder(s) / file(s) to process for the HT path list file contents: 
```

Comment:
Maybe we should be more specific with this line to avoid confusion on how this will work?
Maybe this instruction could say something like
> Select folder(s) whose contents will processed into the HT path list file contents.
> Individual file(s) may also be selected.

The service endpoint's documentation also says the following, which might be helpful, but it's also longer:
>  The paths for folders given in the request will not be included in the results, but if any file paths are included in the request, those paths will also be processed